blog-pipeline/tests/test_llm_config_resolution.py

```
@@ -0,0 +1,249 @@
+# ---------------------------------------------------------------------------
+_fake_config_mod = types.ModuleType("blog_runner.config")
+_fake_config_mod.pipeline_config = MagicMock()
+sys.modules.setdefault("blog_runner.config", _fake_config_mod)
+
+from blog_runner.llm import (
+    get_primary_llm_model,
+    resolve_gateway_model,
+    resolve_llm_execution_target,
+    resolve_llm_model_config,
+    resolve_llm_provider_config,
+)
```

Ruff (0.15.9)

[error] Module level import not at top of file

(E402)